client/components/timeline/timeline-event.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+
+export default class TimelineEvent extends PureComponent {
+	static propTypes = {
+		actionIsBusy: PropTypes.bool,
+		actionIsDisabled: PropTypes.bool,
+		actionIsPrimary: PropTypes.bool,
+		actionIsScary: PropTypes.bool,
+		actionLabel: PropTypes.string,
+		date: PropTypes.instanceOf( Date ),
+		dateFormat: PropTypes.string,
+		detail: PropTypes.string,
+		disabled: PropTypes.bool,
+		icon: PropTypes.string.isRequired,
+		iconBackground: PropTypes.string,
+		onActionClick: PropTypes.func,
+	};
+
+	static defaultProps = {
+		actionIsPrimary: false,
+		dateFormat: 'DD MMMM YYYY',
+		disabled: false,
+		iconBackground: 'info',
+	};
+
+	render() {
+		const {
+			actionIsBusy,
+			actionIsDisabled,
+			actionIsPrimary,
+			actionIsScary,
+			actionLabel,
+			date,
+			dateFormat,
+			detail,
+			disabled,
+			icon,
+			iconBackground,
+			onActionClick,
+		} = this.props;
+		const cardClasses = classNames( 'timeline-event', { 'is-disabled': disabled } );
+		const iconClasses = classNames( 'timeline-event__icon', iconBackground );
+
+		return (
+			<CompactCard className={ cardClasses }>
+				<div className="timeline-event__icon-wrapper timeline-event__column">
+					<div className={ iconClasses }>
+						<Gridicon icon={ icon } size={ 18 } />
+					</div>
+				</div>
+				<div className="timeline-event__main-message timeline-event__column">
+					<div className="timeline-event__title">{ moment( date ).format( dateFormat ) }</div>
```

Comment:
For the next iteration of this, should the date be localized in addition to applying a format?

Reply:
Yep.  I meant to add that.

Reply:
Though I did notice that @fditrapani specifically set the format to `DD MMMM YYYY` instead of using `ll` as I had originally used.

I'll update to use `withLocalizedMoment` for now and leave the default date format as is.